lib/sing_for_needs_web/router.ex

```diff
@@ -10,6 +10,7 @@ defmodule SingForNeedsWeb.Router do
   end
 
   pipeline :api do
+    plug CORSPlug, origin: ["http://localhost:4000", "http://127.0.0.1:4000"]
```

Comment:
This should be on port 3000, because the requests are coming from there, the frontend.